app/views/application/_footer.html.erb

```diff
@@ -36,7 +36,7 @@
             </ul>
         </div>
         <div class="col-md-6 col-lg-3 footer-col" id="footer-services">
-          <h2><%= link_to "Services", services_path, class: "col-heading" %></h2>
+          <h2><%= link_to "Services", service_path(2), class: "col-heading" %></h2>
```

Comment:
Your hard coding a link to an individual service?

Reply:
That was the request.